ArduPlane/tailsitter.cpp

```diff
@@ -13,7 +13,7 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 /*
-  control code for tailsitters. Enabled by setting Q_FRAME_CLASS=10 
```

Comment:
These unrelated whitespace changes should be removed.  They make it harder to trace history.

Reply:
I've removed the whitespace fixes now.

Reply:
CI has passed